.env.example

```
@@ -1,91 +1,109 @@
-HARDHAT_URL = 'http://hardhat'
-HARDHAT_PORT = '8545'
-HARDHAT_PRIVATE_KEY = '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80'
+HARDHAT_URL="http://hardhat"
+HARDHAT_PORT="8545"
+HARDHAT_PRIVATE_KEY="0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
 
 # Consensus Parameters
-CONSENSUS_CONTRACT_ADDRESS = '0x0000000000000000000000000000000000000000'
-DEFAULT_NUM_INITIAL_VALIDATORS = 5
-DEFAULT_CONSENSUS_MAX_ROTATIONS = 3
+DEFAULT_VALIDATORS_COUNT="5"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)